clients/web/src/views/App.js

```diff
@@ -221,7 +223,8 @@ var App = View.extend({
 
             settings = _.extend(settings, {
                 el: this.$('#g-app-body-container'),
-                parentView: this
+                parentView: this,
+                brandName: this.brandName
```

Comment:
This will cause the `brandName` to be passed to every single body view. While this solves the issue at hand, I think it might be a good idea to introduce an application settings registry to handle this and other data (some of which may not be view parameters).

One thing we could do for now is to pass this parameter now as `brandName`, and note that it's unstable and subject to being moved.

@girder/developers What do you all think?